team05_senior_project/build/resources/main/static/like_button.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const e = React.createElement;
+
+class LikeButton extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { liked: false };

```

Comment:
I would move this outside of the constructor. Define it like: `state = {liked: false}` and then if you wanted to set state based on your props or something I would do that inside of the constructor.